frontend/views/UserProfile.vue

```diff
@@ -87,6 +87,16 @@
             </div>
           </div>
         </div>
+        <div class="panel">
+          <p class="panel-heading">
+            <i18n>Set theme</i18n>
+          </p>
+          <div class="panel-block">
+            <select name="" id="" @change="setTheme($event)">
+              <option :value="color.name" v-for="color in colors" v-bind:key="color.name">{{color.name}}</option>
+            </select>
+          </div>
+        </div>
```

Comment:
Going here by default it says the theme is blue, but in fact it is black (although that needs to be fixed too, per the other comment).

Reply:
This is fixed